airbyte-cdk/python/setup.py

```diff
@@ -36,7 +36,7 @@
     name="airbyte-cdk",
     # The version of the airbyte-cdk package is used at runtime to validate manifests. That validation must be
     # updated if our semver format changes such as using release candidate versions.
-    version="0.73.0",
+    version="0.73.1",
```

Comment:
no need to bump the version. this is done automatically by the publish github action

Reply:
Fixed